db/migrate/20230713105553_create_user.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+      t.integer :postsCounter
+
```

Comment:
[OPTIONAL] It is always best to follow the Rails naming convention for table names. In this case, it would be best if you used snake case instead of camel case for the posts counter, comments counter, and likes counter. i.e  `posts_counter`, `comments_counter`, and `likes_counter`.